HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -2100,6 +2100,30 @@ def self.apply_hot_water_distribution(hpxml_bldg, cfa, ncfl, has_uncond_bsmnt)
         hot_water_distribution.recirculation_pump_power = HotWaterAndAppliances.get_default_recirc_pump_power()
         hot_water_distribution.recirculation_pump_power_isdefaulted = true
       end
+      schedules_file_includes_recirculation_pump = (schedules_file.nil? ? false : schedules_file.includes_col_name(SchedulesFile::ColumnHotWaterRecirculationPump))
+      if hot_water_distribution.recirculation_control_type != HPXML::DHWRecirControlTypeTemperature
```

Comment:
I think it's supposed to be:
* DHWRecirControlTypeNone = constant 0.042
* DHWRecirControlTypeTemperature = Temperature Controlled Recirculation Pump
* DHWRecirControlTypeSensor / DHWRecirControlTypeManual = Demand Control Recirculation Pump

DHWRecirControlTypeTimer uses what schedule? The constant 0.042?

Reply:
Also, fix spelling of "Recir" -> "Recirc"?

Reply:
> DHWRecirControlTypeTimer uses what schedule? The constant 0.042?

If you look at the ANSI 301 table, it treats timer control and no control the same. So yes, same schedule. And yes, please fix the typo.